web/app/uusioppija/UusiAmmatillisenKoulutuksenSuoritus.jsx

```diff
@@ -9,13 +9,37 @@ import SuoritustapaDropdown from './SuoritustapaDropdown'
 import Text from '../i18n/Text'
 import {setPeruste} from '../suoritus/PerusteDropdown'
 import Peruste from './Peruste'
+import PaikallinenKoulutusModuuli from './PaikallinenKoulutusModuuli'
 
 export default ({suoritusAtom, oppilaitosAtom, suorituskieliAtom}) => {
-  const suoritustyypitP = koodistoValues('suorituksentyyppi/ammatillinentutkinto,nayttotutkintoonvalmistavakoulutus,ammatillinentutkintoosittainen,valma,telma')
+  const suoritustyypitP = koodistoValues('suorituksentyyppi/ammatillinentutkinto,nayttotutkintoonvalmistavakoulutus,ammatillinentutkintoosittainen,valma,telma,tutkinnonosaapienemmistäkokonaisuuksistakoostuvasuoritus')
```

Comment:
Tähän koodiarvoon on erillisessä haarassa tehty korjaus (poistetaan virheellinen ääkkönen) – tulee huomioida viimeistään kun rebasetetaan masterin päälle.

Reply:
Koodiarvoon ei tullut ääkkösten poistoa, kun kysyin tästä niin Pasi kommentoi että:

"Koodistopalvelu kyllä tukee sitä että koodiarvoissa on ääkkösiä, näitä on aika monessa koodistossa ennestäänkin."

Koodiarvojen tulee tässä matchata suorituksentyyppi.json-tiedostossa oleviin, eli ääkkösten pitänee jäädä ennalleen.